students/hyska_monika/lesson_11_argparse_math/filling_in_the_gaps.py

```diff
@@ -0,0 +1,55 @@
+"""
+Program finds all files with a given prefix, such as spam001.txt,
+spam002.txt, and so on, in a single folder and locates any gaps
+in the numbering (such as if there is a spam001.txt and spam003.txt but no
+spam002.txt). Have the program rename all the later files to close this gap.
+"""
+import shutil
+import os
+import re
+import argparse
+import sys
+
+
+def check_arg(args=None):
+    parser = argparse.ArgumentParser(description='Search files for selected '
+                                                 'prefix and change names')
+    parser.add_argument('-p', '--path',
+                        help='path to search files',
+                        default='.')
+    parser.add_argument('-pr', '--prefix',
+                        help='search file with put prefix, prefix eg.: '
```

Comment:
You should mention that you assume that the files have 3-digit numbers and maybe just let the user choose the beginning of the prefix.